deployments/kernel-net-demo/pod-tc4.yaml

```diff
@@ -0,0 +1,24 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: testpod4
+  labels:
+    env: test
+  annotations:
+      kubernetes.v1.cni.cncf.io/networks: sriov-net-a sriov-net-a
```

Comment:
Is it intended to have blank space between crd names rather than comma?
it looks like this doesn't work when creating the pod.

Reply:
It's meant to be comma separated. Will correct this